src/renderer/styles/ControllersHub.less

```
@@ -0,0 +1,604 @@
+// 本文件经非 scoped <style> 引入，前缀 chub- 全局唯一。
+// ========================================
+
+@font-face {
+  font-family: FusionPixel;
+  src: url('/fonts/fusion-pixel-12px-proportional-zh_hans.woff2') format('woff2');
+  font-weight: 400;
+  font-display: swap;
+}
+
+.chub-page {
+  --pix-font: FusionPixel, PixelMplus12, YouYuan, 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected "FusionPixel" to be "fusionpixel" (value-keyword-case)

(value-keyword-case)